<launch>
  <arg name="camera1_topic" default="/pointgrey/image_rect_color_scaled" />
  <arg name="camera2_topic" default="/nerian/left_image" />
  <arg name="car" default="dut18d" />
  <node pkg="tensorrt_ros" type="tensorrt_ros" name="tensorrt_node" output="screen" >
    <param name="calib_path" value="launch/calib_imgs" />
    <param name="onnx_path" value="yolo_config/yolo/yolo_800320_kmeans_no_leaky.onnx" />
    <param name="trt_path" value="yolo_config/yolo/yolo_800320_kmeans_no_leaky_fp16.trt" />
    <param name="keypoints_onnx_path" value="yolo_config/keypoints/keypoints_paper_spec_seed17.onnx" />
    <param name="keypoints_trt_path" value="yolo_config/keypoints/keypoints_paper_spec_seed17_fp16.trt" />
    <param name="camera1_topic" value="$(arg camera1_topic)" />
    <param name="camera2_topic" value="$(arg camera2_topic)" />
    <param name="car_type" value="$(arg car)" />
    <rosparam>
        yolo_width: 800
        yolo_height: 320
        yolo_classes: 80
        yolo_detection_threshold: 0.5
        yolo_nms_threshold: 0.01
        keypoints_width: 80
        keypoints_height: 80
        max_boxes: 100
        box_min_size_ratio: 0.012
        use_int8: false 
    </rosparam>
  </node>
</launch>

